packages/mdc-textfield/README.md

```diff
@@ -279,6 +279,19 @@ text, validation, and dense UI.
 </div>
 ```
 
+### Sass Mixins
+
+To customize a Text Field's border radii, you can use the following mixins.
+
+#### `mdc-text-field-corner-radius($position, $radius)`
+
+This mixin is provided for customizing the border radius for a corner of a Text Field at the given position,
+e.g.: `@include mdc-text-field-corner-radius(top-left, 10px)` will apply `border-top-left-radius: 10px` to that Text Field.
```

Comment:
don't document what the mixin will output. These are implementation details.

Reply:
I was using this as shorthand to demonstrate that the positional argument should be one of the four possible positions in css, e.g.: top-left, top-right, bottom-left, bottom-right

Open to changing this though, as it seems we will take the approach of making the mixin more component specific, than a general purpose border radius mixin